lib/utils.js

```diff
@@ -20,7 +20,14 @@ const labelParser = function (labels) {
 }
 /* Fingerprinting */
 const shortHash = require('short-hash')
-const fingerPrint = function (text, hex) {
+const xxh = require('xxhashjs')
+const fingerPrint = function (text, hex, alg) {
+  alg = alg || process.env.HASH || 'short-hash'
+  switch (alg) {
+    case 'xxhash64':
+      if (hex) return xxh.h64(text, '').toString(16)
+      else return BigInt(xxh.h64(text, '').toString())
+  }
   if (hex) return shortHash(text)
   else return parseInt(shortHash(text), 16)
```

Comment:
as I mentioned in the issue this also can be improved. [Currently it expands to](https://github.com/joakimbeng/short-hash/blob/b7e1178b45b06bbf1697ba428b7da53664e96979/src/index.js#L5):
```
parseInt(hash(text).toString(16), 16)
```

There's no reason to do that, [hash-string](https://www.npmjs.com/package/hash-string) library can be directly used